lighthouse-cli/test/smokehouse/dobetterweb/dbw-expectations.js

```diff
@@ -185,6 +185,20 @@ module.exports = [
           },
         },
       },
+      'image-aspect-ratio': {
```

Comment:
dbw seemed like the best place for this since it's already running all of Best Practices, but happy to move